### ISSUE TITLE
result of Substitute is ignored

### ISSUE THREAD

**Issue by olafhering** [openSUSE/open-build-service#5260]
I have a '`Substitute: pkgconfig(pkg) pkg-devel=`' in prjconf.
The trailing '=' is supposed to carry the version during substitution.
The buildlog shows that BuildRequires is properly updated, but the result is apparently not used when preparing the build. In my case I expect state '`unresolvable`' because the expected version is not available in the target repository. As a result the pkg starts to build, then rpmbuild fails:
````
...
[   39s] [112/251] installing python-devel-2.6.9-6.SLE_11.1
...
[   83s] < BuildRequires:  pkgconfig(python2) >= 2.7
[   83s] > BuildRequires:  python-devel >= 2.7
...
[   83s] error: Failed build dependencies:
[   83s] 	python-devel >= 2.7 is needed by ${pkg}
...
````

I think such "Substitute" used to bring a pkg in state '`unresolvalbe`' in the past.

**Comment by mlschroe** [comment]
No, it never did that. But maybe it's a good idea to use the "versioned" substitutes when expanding package dependencies to get "unresolvable" instead of "failed" as build result.
